pkg/shell/nav.jsx

```diff
@@ -30,6 +30,11 @@ export const SidebarToggle = () => {
     );
 };
 
+function uncalled_function() {
+    console.log("This whole function should not be called");
+    console.log("And another line to make a bigger code block");
+}
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17475-20220628-083430-8f8981f7-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)